www/blocks-view.php

```diff
@@ -25,6 +25,10 @@
 
 	$no_of_records_per_page = 20;
 	$offset = ($page-1) * $no_of_records_per_page;
+
+	#what this must be?
+	$search = "???";
+
```

Comment:
I don't understand the question. $offset is the query row offset.

There's no search feature on blocks-view.php. Just click the IP you want detail on.

Reply:
> I don't understand the question. $offset is the query row offset.
> 
> There's no search feature on blocks-view.php. Just click the IP you want detail on.

The $ search variable was not declared / initialized and was used below in a message, it threw an error for me, so I "initialized" and marked this comment for you to check if this is correct or if the variable was being used in the message by mistake.